crossmodal/parsers/aff_parser/AffParser.py

```diff
@@ -0,0 +1,60 @@
+import glob
```

Comment:
All parsers should be working for only one file! This should be renamed to Dataset PArser it should be Specific for IEMOCAP and also using any other parses such as Audio Parser, Text Parse, etc.